test/testParachainStaking.t.sol

```diff
@@ -201,21 +346,47 @@ contract ParachainStakingTest is Test {
         assertEq(token.balanceOf(address(staking)), 20);
         assertEq(token.balanceOf(address(paraOwner)), 80);
 
-        // Try withdraw stake before lock period expires
-        vm.expectRevert("lock period not expired");
-        staking.withdrawParachainStake(fakeParaId);
-
-        // Wait for lock period to expire (7 days after staker start date)
-        vm.warp(block.timestamp + 7 days + 1 seconds);
         // Withdraw stake
         staking.withdrawParachainStake(fakeParaId);
         assertEq(token.balanceOf(address(staking)), 0);
         assertEq(token.balanceOf(address(paraOwner)), 100);
 
         vm.stopPrank();
+
+        // todo: how to test transfer fail?
```

Comment:
This should in theory never fail if our contract is built correctly. You could however use a custom token contract which allows you to edit a given address' balance, and then lower the contract's bal below the transfer amount.

Reply:
Removed